### PR TITLE
add splitting step to geometric notebook

### LINKED ISSUE #67: Split geometric into train and testsets.
- Update the zip file
- Ping whoever was working on the model generating scripts so they can be rerun.

### PR DESCRIPTION
**Pull request by cwmeijer**
single commit. added splitting into train/test at the end of the script

### CODE REVIEW

File: example_data/dataset_preparation/geometric_shapes/generate_geometric_shapes.ipynb [hunk 1]
```diff
@@ -2,7 +2,7 @@
  "cells": [
```
Comment (loostrum) [inline example_data/dataset_preparation/geometric_shapes/generate_geometric_shapes.ipynb:2]: <p>Nice to put this in a separate cell! I'm thinking now we could also create the original shapes array with dtype float (which it might be already, as the dtype wasn't specified and I think float is the default). I'd think doing <code>shapes /= 255.</code>  would then have a much lower load.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (loostrum) [inline example_data/dataset_preparation/geometric_shapes/generate_geometric_shapes.ipynb:2]: <p>Could also do <code>X_train, X_test, y_train, y_test = train_test_split(shapes, labels, ....)</code> . Perhaps add in a <code>random_state</code>  as well for reproducibility. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (loostrum) [inline example_data/dataset_preparation/geometric_shapes/generate_geometric_shapes.ipynb:2]: <p>I think it was nice to have the figure with all the shapes here, maybe it could be put back?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

**Comment by loostrum** [comment]
Hmm I tried to review through reviewnb but it doesn't post the comments here in a sensibile location in the notebook. There are in the correct location on the reviewnb website so you could have a look there.

Overall it's a nice addition to the code 👍

**Review by loostrum — APPROVED** [review]

File: example_data/dataset_preparation/geometric_shapes/generate_geometric_shapes.ipynb (hunk 1 above)
Comment (cwmeijer) [reply example_data/dataset_preparation/geometric_shapes/generate_geometric_shapes.ipynb:2]: <p>I agree, it was great. Isn't it still there though? I can still see the code block and it even looks unchanged. Also, I remember having checked the overview image after running the notebook with the changes to make sure everything is still alright. Maybe I forgot to run the cell before saving it? I'll check.</p>

Comment (loostrum) [reply example_data/dataset_preparation/geometric_shapes/generate_geometric_shapes.ipynb:2]: Weird, when I clone the branch and open jupyter lab, I see a big white space about the size of the figure. So it's there, but not really? 🤔

Comment (cwmeijer) [reply example_data/dataset_preparation/geometric_shapes/generate_geometric_shapes.ipynb:2]: Makes sense, but I leave it like it is anyways :-P Good enough as it is I'd say.

Comment (cwmeijer) [reply example_data/dataset_preparation/geometric_shapes/generate_geometric_shapes.ipynb:2]: I now set a seed in cell 1 but also just before splitting into sets. A bit overkill maybe, but I think it's good.

Comment (cwmeijer) [reply example_data/dataset_preparation/geometric_shapes/generate_geometric_shapes.ipynb:2]: I think it's back now.
